src/partr.c

```diff
@@ -373,10 +373,11 @@ JL_DLLEXPORT void jl_set_task_tid(jl_task_t *task, int tid) JL_NOTSAFEPOINT
 }
 
 // get the next runnable task from the multiq
-static jl_task_t *get_next_task(jl_value_t *getsticky)
+static jl_task_t *get_next_task(jl_value_t *trypoptask, jl_value_t *q)
 {
     jl_gc_safepoint();
-    jl_task_t *task = (jl_task_t*)jl_apply(&getsticky, 1);
+    jl_value_t *args[2] = { trypoptask, q };
+    jl_task_t *task = (jl_task_t*)jl_apply(&args, 2);
```

Comment:
The compiler says there's too many levels of pointer indirection here:
```
/data/vtjnash/julia1/src/partr.c: In function ‘get_next_task’:
/data/vtjnash/julia1/src/partr.c:380:44: warning: passing argument 1 of ‘jl_apply’ from incompatible pointer type [-Wincompatible-pointer-types]
     jl_task_t *task = (jl_task_t*)jl_apply(&args, 2);
                                            ^
In file included from /data/vtjnash/julia1/src/partr.c:8:0:
/data/vtjnash/julia1/src/julia.h:1642:13: note: expected ‘jl_value_t ** {aka struct _jl_value_t **}’ but argument is of type ‘jl_value_t * (*)[2] {aka struct _jl_value_t * (*)[
2]}’                                
 jl_value_t *jl_apply(jl_value_t **args, uint32_t nargs)
```

Should be `&args[0]` or just `args`.